cmd/thanos/query.go

```diff
@@ -146,6 +146,8 @@ func registerQuery(app *extkingpin.App) {
 	enableMetricMetadataPartialResponse := cmd.Flag("metric-metadata.partial-response", "Enable partial response for metric metadata endpoint. --no-metric-metadata.partial-response for disabling.").
 		Hidden().Default("true").Bool()
 
+	featureList := cmd.Flag("enable-feature", "Comma separated experimental feature names to enable.The current list of features is promql-negative-offset and promql-at-modifier.").Default("").Strings()
```

Comment:
The current flag is a string list instead of a comma-separated string. @metonymic-smokey Which option would u like to go with?

Reply:
@yeya24 I would prefer the current option since here, entering a comma separated list of  features parses it into a list, which saves an additional step here.

Reply:
If u prefer the current option, then please update the description as we are not using a comma separated string. We are using repeated flags here

Reply:
@yeya24 the comma separated values result in a slice of strings.

Reply:
```suggestion
	featureList := cmd.Flag("enable-feature", "Comma separated experimental feature names to enable. The current list of features is 'promql-negative-offset' and 'promql-at-modifier'.").Default("").Strings()
```